plugins/flytekit-papermill/dev-requirements.in

```diff
@@ -1 +1,2 @@
-flytekitplugins-spark>=0.30.0b4
+git+https://github.com/flyteorg/flytekit@add-sd-make-class-methods#egg=flytekitplugins-spark&subdirectory=plugins/flytekit-spark
```

Comment:
this should be moved to `master` in a follow-up PR?

Reply:
yeah, i'll do as part of my next pr.